src/main/java/me/jellysquid/mods/sodium/client/gui/SodiumOptionsGUI.java

```diff
@@ -198,7 +197,7 @@ private void renderOptionTooltip(MatrixStack matrixStack, ControlElement<?> elem
         OptionImpact impact = option.getImpact();
 
         if (impact != null) {
-            tooltip.add(Language.getInstance().reorder(new LiteralText(Formatting.GRAY + "Performance Impact: " + impact.toDisplayString())));
+            tooltip.add(Language.getInstance().reorder(new TranslatableText("sodium.options.performance_impact_string", "", impact.getText()).formatted(Formatting.GRAY)));
```

Comment:
```suggestion
            tooltip.add(Language.getInstance().reorder(new TranslatableText("sodium.options.performance_impact_string", impact.getText()).formatted(Formatting.GRAY)));
```
I think this empty string can be removed, let me know if I am wrong.

Reply:
It can be removed, I just kept it since I didn't want to touch the translation strings.

Reply:
Although this should be update in OneSky, further discussion might be required on how Jelly plans to handle.

Reply:
It looks like this may have already been addressed in [`f8c3a67`](https://github.com/CaffeineMC/sodium-fabric/commit/f8c3a67822ca594ee934d5da0fd12097f4daf5d1) (made after the original PR was merged into dev), as the empty string has been removed there. The translation keys on OneSky also seem to have the empty string removed, so it should be safe to remove.

Reply:
Nice catch, I will commit the changes.